src/pykeen/datasets/cli.py

```diff
@@ -32,13 +28,39 @@ def summarize():
             click.secho(str(e), fg='red', bold=True)
 
 
+def _iter_datasets():
+    from . import datasets
+    import docdata
+    it = tqdm(
+        sorted(
+            datasets.items(),
+            key=lambda pair: docdata.get_docdata(pair[1])['statistics']['triples'],
+        ),
+        desc='Datasets',
+    )
+    for k, v in it:
+        it.set_postfix(name=k)
+        yield k, v
+
+
 @main.command()
 @verbose_option
-@click.argument('dataset')
+@click.option('--dataset')
 @click.option('-f', '--force', is_flag=True)
+@click.option('-a', '--all-datasets', is_flag=True)
```

Comment:
Extend `--dataset` option to allow for (simple) regex instead? 

e.g. 
```console
$ python -m pykeen.datasets.cli --dataset "*"
```
to run it for all datasets, and
```console
$ python -m pykeen.datasets.cli --dataset "wn18rr"
```
for WN18RR, etc.

Reply:
Good suggestion. Done in a5228098